modules/stochastic_tools/src/transfers/SamplerParameterTransfer.C

```diff
@@ -77,32 +82,122 @@ SamplerParameterTransfer::executeToMultiapp()
 {
   if (getToMultiApp()->isRootProcessor())
   {
-    SamplerReceiver * ptr = getReceiver(_app_index);
-    ptr->transfer(_parameter_names, _row_data);
+    // Get the parameters in command-line format
+    const std::string cmd_line =
+        SamplerFullSolveMultiApp::sampledCommandLineArgs(_row_data, _parameter_names);
+
+    // Get the sub-app SamplerReceiver objects and transfer param-values map
+    for (auto & ptr : getReceivers(_app_index, cmd_line))
```

Comment:
@GiudGiud I think this gets the parameter receivers in the subapps recursively so it sends the new values directly.

Reply:
As far as I understand, for cases which don't need the controllable parameters, we can use the command line with the `:` separators for subapps.

Reply:
oh ok it s the whole purpose of the PR

Reply:
I think it's fine (have not reviewed), we probably should talk about it to get in synch though